### PR TITLE
Search updated

### LINKED ISSUE #27: Search should also return events posted by users with keyword in name and statement

### PR DESCRIPTION
**Pull request by ZFW777**
Search events posted by users with keyword (tested). fix #27 
System Config model updated, but not tested.

### CODE REVIEW

**Review by R-o-y — COMMENTED** [review]
has the change been tested

**Comment by ZFW777** [comment]
Tested.

File: mocacares_backend/event_platform/views/views_event.py [hunk 1]
```diff
@@ -72,7 +78,10 @@ def get_events(request):
     if event_type is not None:
         events = events.filter(event_type=event_type)
     if search_key is not None:
-        events = events.filter(Q(title__icontains=search_key) | Q(description__icontains=search_key))
+        keyword_matching_events = events.filter(Q(title__icontains=search_key) | Q(description__icontains=search_key))
+        keyword_matching_users = User.objects.filter(Q(username__icontains=search_key) | Q(statement__icontains=search_key))
+        events_posted_by_matching_users = events.filter(poster__in=keyword_matching_users)
+        events = keyword_matching_events | events_posted_by_matching_users
```
Comment (R-o-y) [inline mocacares_backend/event_platform/views/views_event.py]: is this operator equivalent to union function?

**Review by R-o-y — COMMENTED** [review]
cool, i will test and merge it tomorrow then

File: mocacares_backend/event_platform/views/views_event.py (hunk 1 above)
Comment (ZFW777) [reply mocacares_backend/event_platform/views/views_event.py]: yes

**Comment by R-o-y** [comment]
maybe use the union function can make the code more explicit?

**Comment by ZFW777** [comment]
OK, will edit

**Pull request merged**